### DISCUSSION TITLE
Compatibility with react native

### DISCUSSION THREAD

**Discussion by lucasAzS** [adobe/react-spectrum#5673] [Q&A] [answered]
Does this libs or parts of it is compatible with react-native, has plans to or is just web?

**Comment by reidbarber** [comment, marked as answer]
The hooks should, but it is unlikely that components work due to various dependencies on `react-dom`. There is a community-made [react-native-aria](https://github.com/GeekyAnts/react-native-aria), but it looks like it has moved over to the [gluestack-ui](https://github.com/gluestack/gluestack-ui) repo. There aren't any plans for wider react-native support that I'm aware of.
